### ISSUE TITLE
Modello Binario

### ISSUE THREAD

**Issue by MrGreensushi** [MrGreensushi/MachineLearningProject#5]
Implementare classe su python per il modello binario

    creare coppie di immagini 
    il modello prendo in ingresso la coppia e in paralello lo manda a 2 CNN
    Distanza euclidea dei 2 cnn

Scelta iniziale degli iperparametri

    Funzione di loss e il valore alpha associato
    numero epoche
    early stop -> parametro da analizzare: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/TopKCategoricalAccuracy
    valore di distanza tra le due immagine di threshold

plottare topk accuracy alla fine
